build.gradle

```diff
@@ -290,3 +309,87 @@ tasks.verifyGoogleJavaFormat {
   exclude 'build/**'
 }
 test.dependsOn verifyGoogleJavaFormat
+
+// Release
+// =======
+
+task updateSamples << {
+  def outputContent = new File('templates/sample_pom.xml.template').text
+  // Yes this is inefficient, but it is very rarely done.
+  outputContent = outputContent.replace('{{gaxVersion}}', project.version)
+  outputContent = outputContent.replace('{{grpcVersion}}', grpcVersion)
+  outputContent = outputContent.replace('{{commonProtosVersion}}', commonProtosVersion)
+  new File('samples/pom.xml').write(outputContent)
+}
+
+task updateReadme << {
+  def quickstartContent = new File('templates/QUICKSTART.md.template').text
+  quickstartContent = quickstartContent.replace('{{gaxVersion}}', project.version)
+  def readmeLines = []
+  def sectionStarted = false
+  def sectionEnded = false
+  new File('README.md').eachLine { String line ->
```

Comment:
I don't think "crazy" is the right adjective - maybe "verbose"? In any case, I adapted your code to the compiler's complaints so that the code is now more concise.